docs/UPGRADING.rst

```diff
@@ -0,0 +1,17 @@
+Upgrading HaaS
+==============
+
+This file describes the procedure for upgrading HaaS to a new version.
+
+1. Read the release notes, as they may contain version-specific information.
+2. Stop the HaaS services (webserver and HaaS network daemon).
+3. Download and install the new version of HaaS::
+
+     $ git clone https://github.com/cci-moc/haas
+     $ python.setup.py install
```

Comment:
Good catch on the typo; I will fix.

There is an uninstall command of some variety, but from what I know it isn't really to be relied upon. This should at least overwrite things, so should be okay, but idally we *would* want to do a proper uninstall. For that though we should be doing rpm/deb packages, as pip/setup.py won't do a reasonable job of it. We should do that at some point.